release/Earthfile

```
@@ -133,6 +107,112 @@ signed-release:
 
     SAVE ARTIFACT ./release AS LOCAL ./release
 
+# promote-release is a single target that should be called after creating a GH release that copies all
+# releasables (images, artifacts) to their target destinations.
+promote-release:
+    BUILD +promote-artifacts-to-gh
+    BUILD +promote-to-dockerhub
+
+gh-cli:
+    FROM alpine:3.20
+
+    RUN apk add --no-cache ca-certificates curl tar jq git bash && update-ca-certificates
+    
+    RUN apk add --no-cache github-cli
```

We should start pinning these also:

```suggestion
    RUN apk add --no-cache github-cli=2.47.0-r4
```